src/analysis/inner_product_space/pi_L2.lean

```diff
@@ -176,6 +177,12 @@ rfl
   ((v.isometry_euclidean_of_orthonormal hv).symm : euclidean_space 𝕜 ι → E) = v.equiv_fun.symm :=
 rfl
 
+@[simp] lemma basis.map_isometry_euclidean_of_orthonormal (v : basis ι 𝕜 E) (hv : orthonormal 𝕜 v)
```

Comment:
Is this actually useful as a `simp` lemma? Wouldn't you just want to call it explicitly?

Reply:
Well, I think the RHS is simpler than the LHS, and I've used it as a `simp` lemma (to prove the corresponding `simp` lemma for the composition of `complex.isometry_euclidean` and `isometry_euclidean_of_orthonormal`, which in turn is used in defining and proving properties of oriented angles). If these aren't `simp` lemmas they can of course be named directly in the relevant `simp` calls.